README.md

```diff
@@ -118,7 +117,7 @@ To run tests, run the following command:
 👤 **Samuel Popoola**
 
 - GitHub: [GitHub](https://github.com/Tobby8629)
-- Twitter: [Twitter](https://twitter.com/tobby_samuels)
+- Twitter: [Twitter](https://twitter.com/thobie_tobeh)
```

Comment:
**[OPTIONAL]**

- [ ] Kindly put your picture there because I notice a thing that your Twitter account doesn't have your picture and a lot of recruiters find professional developers on Twitter as well so if your profile isn't professional then there is a high possibility that they might not connect with you on Twitter. So please put your professional picture there :+1: